packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -25,7 +25,7 @@ const MAX_LOCAL_SECONDARY_INDEX_COUNT = 5;
 /**
  * Options for configuring a system errors metric that considers multiple operations.
  */
-export interface SystemErrorsForOperationsMetricOptions extends cloudwatch.MetricOptions {
+export interface OperationsMetricOptions extends cloudwatch.MetricOptions {
```

Comment:
I don't think we can rename this interface. I think I might prefer having a second input prop for
operations.

```ts
metricThrottledRequestsForOperations(operations: string[], props?: cloudwatch.MetricOptions):
cloudwatch.IMetric;
```

Reply:
I can reverce the interface but I think creating another interface for throttle will be better to keep a pattern

Reply:
I do like the API with the `operations` as a separate input property better, but I also agree with you that it is better to keep the current patter.